core/domain/suggestion_services_test.py

```diff
@@ -443,10 +443,10 @@ def test_check_can_resubmit_suggestion(self):
                     self.target_id, self.target_version_at_submission,
                     self.author_id, self.change, 'test description',
                     self.reviewer_id)
-        can_resubmit = suggestion_services.check_can_resubmit_suggestion(
+        can_resubmit = suggestion_services.check_can_resubmit_suggestion(# pylint: disable=line-too-long
```

Comment:
Why add the pragma? It is the same length as before.

Reply:
ok

Reply:
done